lib/dependencies/inspect-implementation.ts

```diff
@@ -154,6 +159,20 @@ export async function inspectInstalledDeps(
   }
 }
 
+// this enforces PIPENV_PIPFILE if dealing with a Pipfile
+// if PIPENV_PIPFILE is exported/-x and not the same as
+// targetfile we will still fail however
```

Comment:
The code speaks for itself, but if you are adding a comment then maybe:
```suggestion
// For Pipfile, set and export PIPENV_PIPFILE correlated with targetFile path. Otherwise process.env is used.
```

What does this mean? Can we add a test that shows this scenario?
`// if PIPENV_PIPFILE is exported/-x and not the same as`
`// targetfile we will still fail however`